semantic/src/site/globals/site.overrides

```diff
@@ -169,15 +170,58 @@ p strong {
   margin-bottom: 5px;
 }
 
+// Steps line (vertical box - layout)
+#application-area .line-vertical-box {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  min-height: 30px; // TODO: Need to be updated for mobile
+  align-items: center;
+  justify-content: space-between;
+}
+
+// Inner steps line (left side - icon & header)
+#application-area .line-vertical-box .left {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
```

Comment:
You can add `flex-grow: 1` to this to push the progress bar further to the right. It makes this (left) element able to grow (relative to the others) in the parent flexbox

Reply:
I end up doing this a little different, please have a look: d425404

Reply:
The problem is the "right-align-flex-box" isn't wrapping on mobile:
![Screen Shot 2021-04-23 at 1 57 40 PM](https://user-images.githubusercontent.com/5456533/115807121-16a77b80-a43c-11eb-8d61-399a07773ac5.png)

You could add more to make this, work, but you really don't need the `.right-align-flex-box-row` at all -- just add a `flex-grow: 1` property to the left item and it works fine with a lot less complexity.

Reply:
Thanks, I thought I have checked on mobile and it was fine/ But probably not on smaller versions. Have done as you suggested now.